vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/marked_to_no_longer_receive_frequency_cap_unittest.cc

```diff
@@ -41,10 +41,10 @@ TEST_F(BatAdsMarkedToNoLongerReceiveFrequencyCapTest,
        DoNotAllowAdIfMarkedToNoLongerReceive) {
   // Arrange
   CreativeAdInfo ad;
-  ad.creative_set_id = kCreativeSetId;
+  ad.segment = kSegment;
 
-  Client::Get()->ToggleAdThumbDown(ad.creative_instance_id, ad.creative_set_id,
```

Comment:
do we not need to consider the thumb down case anymore? So users can only opt-out of whole segments now and a thumb down will just be ignored?

Reply:
Unfortunately thumbs down does not make a lot of sense right now as it would only temporarily block ads due to the creative set as this would change in the future. However we should raise another issue to consider thumbs down not showing ads from a specific advertiser. Your thoughts?

Reply:
This was a bug as this frequency cap was meant to block whole categories

Reply:
I like the idea of blocking based on advertiser, but never heard users or business requesting it. So no need for a ticket imo

Reply:
Spoke with @jsecretan and thumbs-down should use the advertiser id. I will fix this now in the same issue, however it will not work for legacy history as we don't have the advertiser id in the data structure :-)

Reply:
Thumbs down (dislike) uses `creative_set_id` (raise https://github.com/brave/brave-browser/issues/17319 as an enhancement to use the advertiser id)

Marked to no longer recieve uses the `segment`

Marked as inappropriate uses the `creative_set_id`